tensorflow/python/kernel_tests/conv_ops_3d_test.py

```diff
@@ -81,27 +94,36 @@ def _VerifyValues(self, tensor_in_sizes, filter_in_sizes, stride, padding,
                     expected):
     results = []
     for data_format, use_gpu in GetTestConfigs():
-      result = self._SetupValuesForDevice(
-          tensor_in_sizes,
-          filter_in_sizes,
-          stride,
-          padding,
-          data_format,
-          use_gpu=use_gpu)
-      results.append(result)
-      tolerance = 1e-2 if use_gpu else 1e-5
+      for dtype in self._DtypesToTest(use_gpu):
+        result = self._SetupValuesForDevice(
+            tensor_in_sizes,
+            filter_in_sizes,
+            stride,
+            padding,
+            data_format,
+            dtype,
+            use_gpu=use_gpu)
+        results.append(result)
+
       with self.test_session() as sess:
-        values = sess.run(results)
+        tensor_results = [result[0] for result in results]
```

Comment:
Why is this line necessary? Why take the first element for each result?

Reply:
Sorry, probably it was used for manual comparison of results. 
I fixed it.